eng/pipelines/libraries/helix-queues-setup.yml

```diff
@@ -50,7 +50,7 @@ jobs:
     # Linux musl x64
     - ${{ if eq(parameters.platform, 'Linux_musl_x64') }}:
       - ${{ if or(ne(parameters.jobParameters.isExtraPlatforms, true), eq(parameters.jobParameters.includeAllPlatforms, true)) }}:
-        - (Alpine.314.Amd64.Open)Ubuntu.1804.Amd64.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:alpine-3.14-helix-amd64-20220803180115-99b3286
+        - (Alpine.314.Amd64.Open)Ubuntu.1804.Amd64.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:alpine-3.14-helix-amd64-20220912172439-e7e8d1c
```

Comment:
The previous image had an older version of MsQuic which returned INVALID_STATE instead of ALPN_IN_USE

Reply:
Is that in 2.1+ or just in msquic `main` @rzikm?

Reply:
The change is in 2.1